jobs/build/tarball-sources/Jenkinsfile

```
@@ -142,9 +143,24 @@ node {
                 }
                 echo "sucessfully run cmd: ${jira}"
             }
+
+            stage("slack notification to release channel") {
+                jiraCardURL = "https://projects.engineering.redhat.com/browse/${jira.Key}"
+
+                slacklib.to(version).say("""
+                *:heavy_check_mark: tarball-sources sent to CLOUDDST*
+                CLOUDDST JIRA: ${jiraCardURL}
+                buildvm job: ${commonlib.buildURL('console')}
+                """)
+            }
         }
 
     } catch (err) {
+        slacklib.to(version).say("""
+        *:heavy_exclamation_mark: tarball-sources failed*
```

seems likely to get lost in the noise if we don't call out... we do want some action to be taken here
```suggestion
        *:heavy_exclamation_mark: @release-artists tarball-sources failed*
```